src/projects/UoM.story.tsx

```diff
@@ -0,0 +1,672 @@
+import React from "react";
+import {
+  Box,
+  Flex,
+  Heading2,
+  Heading3,
+  Text,
+  Icon,
+  QuietButton,
+  PrimaryButton,
+  Table,
+  DescriptionList,
+  DescriptionGroup,
+  DescriptionTerm,
+  DescriptionDetails,
+  Tooltip,
+  Select,
+  Input,
+  Toggle,
+  IconicButton,
+  TruncatedText,
+} from "../index";
+import { Columns } from "../Table/Table.types";
+import styled from "styled-components";
+import { border, minWidth, width } from "styled-system";
+
+export default {
+  title: "Projects/UoM",
```

Comment:
Suggestion: create a more specific hierarchy for the page, something like "Projects/ShopFloor/2025/UoM